napari/_qt/qt_viewer.py

```diff
@@ -212,12 +212,14 @@ def screenshot(self):
 
     def _open_images(self):
         """Add image files from the menubar."""
-        filenames, _ = QFileDialog.getOpenFileNames(
-            parent=self,
-            caption='Select image(s)...',
-            directory=self._last_visited_dir,  # home dir by default
-        )
-        self._add_files(filenames)
+        open_dialog = QFileDialog()
+        open_dialog.setWindowTitle('Select file(s) or folder(s)')
+        open_dialog.setDirectory(self._last_visited_dir)
```

Comment:
you actually need to add
```python
        open_dialog.setFileMode(QFileDialog.ExistingFiles)
```
to be able to select multiple files at the same time. i have verified that works, and selecting multiple images of the same shape does the right thing

Reply:
oh wait no - adding that blocks the ability to open the folders :-(